GameData/ROEngines/PartConfigs/RSRMNose_PC.cfg

```diff
@@ -0,0 +1,130 @@
+PART
+{
+	module = Part
+	name = ROE-RSRMNose
+	author = DylanSemrau
+
+	category = Engine
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+
+	mass = 4.503
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
+
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+
+	MODEL
+	{
+		model = ROEngines/Assets/DylanSemrau/RSRMNose
+		position = 0.0, 2.09556, 0.0
+	}
+
+	MODEL
+	{
+		model = ROEngines/Assets/DylanSemrau/RSRMAvionics
+	}
+
+	scale = 1
+	rescaleFactor = 1.65
+	node_stack_Bottom = 0.0, -1.0227, 0.0, 0.0, -1.0, 0.0, 2
+	CoMOffset = 0.0, 0.897631, 0.0
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,0,1,0,0
+
+	title = RSRM Nose Cone
+	manufacturer = Thiokol & Orbital ATK
+	description = Part of the Shuttle Solid Rocket Booster, this nose cone contains the separator motors to help prevent a collision after separation.
+
+	tags = thiokol, solid, shuttle, 4 segment, rsrm, orbital atk, sls, 5 segment, rsrmv
+
+	stagingIcon = SOLID_BOOSTER
+	stageOffset = 1
+	childStageOffset = 1
+
+	skinInternalConductionMult = 4.0
+	emissiveConstant = 0.5
+	dragModelType = default
+	maximum_drag = 0.3
+	minimum_drag = 0.2
+	angularDrag = 2
+	breakingForce = 200
+	breakingTorque = 200
+	crashTolerance = 7
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = thrustTransform
+		EngineType = SolidBooster
+		throttleLocked = True
+		exhaustDamage = False
+		allowShutdown = False
+		ignitionThreshold = 0.1
+		minThrust = 430.4
+		maxThrust = 430.4
+		heatProduction = 100
+		PROPELLANT
+		{
+			name = HTPB
+			ratio = 1.0
+			DrawGauge = True
+		}
+		ullage = False
+		pressureFed = False
+		ignitions = 1
+		atmosphereCurve
+		{
+			key = 0 240.9
+			key = 1 240.9
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = HTPB
+		volume = 78.6
+		basemass = -1
+
+		TANK
+		{
+			name = HTPB
+			amount = 78.6
+			maxAmount = 78.6
+		}
+	}
```

Comment:
We can do this, but should it be a proper engine config in RO instead of declared in the part?